lib/output/util/format_type.js

```diff
@@ -118,7 +118,10 @@ function formatType(getHref/*: Function*/, node/*: ?Object */) {
   case Syntax.NameExpression:
     return [link(node.name, getHref)];
   case Syntax.ParameterType:
-    return [t(node.name + ': ')].concat(formatType(getHref, node.expression));
+    if (node.name) {
+        result.push(t(node.name + ': '));
```

Comment:
So this prints `': T'` for the type `number => string`. Maybe it should just print `T`?